app/main.py

```diff
@@ -1,7 +1,22 @@
-def get_weather() -> None:
-    # write your code here
-    pass
+import os
+import requests
+
+
+def get_weather(query: str) -> None:
+    print("Performing request to Weather API for city Paris...")
+    api_key = os.environ.get(
+        "API_KEY",
+        "45aa589d47f145f3874211504242703"
```

Comment:
remove this `API_KEY`

better to check, if api_key is None -> raise some error